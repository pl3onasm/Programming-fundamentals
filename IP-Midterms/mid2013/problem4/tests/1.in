5 6
1
1 0 5
