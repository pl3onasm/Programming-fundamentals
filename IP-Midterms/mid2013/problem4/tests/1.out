000000
111110
000000
000000
000000
